cms/src/plugins/string-array/admin/src/components/Input/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { useIntl } from 'react-intl';
+import { Textarea, Typography } from '@strapi/design-system';
+
+const Input = ({
+  attribute,
+  error,
+  name,
+  onChange,
+  value
+}) => {
+  const { formatMessage } = useIntl();
+  const [arrayOfStrings, setArrayOfStrings] = useState(() => {
+    return Array.isArray(value) ? value.join(',') : value;
+  });
+
+  const handleValueChange = (newValue) => {
+    setArrayOfStrings(newValue);
+    onChange({ target: { name, value: newValue, type: attribute.type } });
+  }
```

Comment:
```suggestion
  const handleValueChange = useCallback((e) => {
    const newValue = e.target.value;
    setArrayOfStrings(newValue);
    onChange({ target: { name, value: newValue, type: attribute.type } });
  }, []);
```
This form will avoid extra assignations in the renderization cycle loop of React